sdk/servicebus/service-bus/src/connectionContext.ts

```diff
@@ -457,6 +496,9 @@ export namespace ConnectionContext {
           connectionError || contextError,
           "streaming"
         );
+        // TODO: What to do for "disconnect" on sessions streaming receiver?
```

Comment:
What do we do for non-session streaming receivers?

Reply:
Need to check.. I guess it just hangs.
And the current plan is to close the streaming receiver after notifying through `processError`.

Reply:
Confirming.. this just hangs as of now.
In a future PR.. I'll handle the "close the streaming receiver after notifying session-lock-lost through `processError`".

Reply:
> Need to check.. I guess it just hangs.
And the current plan is to close the streaming receiver after notifying through processError.

That is for session streaming case right? Chris's question was for non session streaming receiver

Also, for sessions, we don't have separate underlying batching and streaming receivers like we do for non sessions

Reply:
Oops, misread.
Streaming receiver non-sessions recovers just fine.